src/PKSim.Presentation/Presenters/Simulations/GlobalPKAnalysisPresenter.cs

```diff
@@ -167,15 +173,13 @@ public bool HasParameters()
 
       public bool CanCalculateGlobalPK()
       {
-         return firstSimulation.Compounds.Any(compound =>
-         {
-            var schemaItems = _protocolToSchemaItemsMapper.MapFrom(firstSimulation.CompoundPropertiesFor(compound).ProtocolProperties.Protocol);
-            return !isMultipleIV(schemaItems);
```

Comment:
@PavelBal @Yuri05 
Can someone explain to me why the condition to calculate global pk is to not have multiple IV?
Multiple oral is fine, mix of oral and iv is fine, but two multiple no? I don't get it

This also comes to the next error that @rwmcintosh is looking at where we have user defined stuff going on. Feels like this was not specified properly

Reply:
Yep, I think `!isMultipleIV` is wrong. Single_IV, Single_Oral and Multiple_Oral must be allowed, anything else probably not.
In the individual simulations:
* no global analysis is shown for IV_Oral_Mix, IV_UserDef_Mix etc.
* global analysis is shown for UserDef Single - which is wrong I think. All calculated values are then NaN, Infinity, Zero etc.
![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/f4a2577c-0703-44a7-8893-0eddcfbe6ffb) ![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/230ac3a3-9308-4f04-96a2-c5b701d6ea67)

So I think it makes sense to show global analysis only for the 3 scenarios:
* IV single
* Oral single
* Oral Multiple

Reply:
Sounds good. Agree with this and I'll make an update for this